autopark/Vehicle.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace autopark
+{
+    enum Color
+    {
+        Blue = 1,
+        White,
+        Green,
+        Gray,
+        Yellow,
+        Red
+    }
```

Comment:
Move Enum to separate file.

Reply:
![image](https://user-images.githubusercontent.com/79978383/115912422-f8f80780-a477-11eb-8458-94ba7ae8ac49.png)
Done